moto/cloudfront/models.py

```diff
@@ -315,9 +316,20 @@ def update_distribution(self, dist_config, _id, if_match):
     def create_invalidation(self, dist_id, paths, caller_ref):
         dist, _ = self.get_distribution(dist_id)
         invalidation = Invalidation(dist, paths, caller_ref)
+        try:
+            self.invalidations[dist_id].append(invalidation)
+        except KeyError:
+            self.invalidations[dist_id] = [invalidation]
 
         return invalidation
 
+    def list_invalidations(self, dist_id, marker, _max_items):
+        if marker:
+            raise NotImplementedError(
+                "moto: paging of invalidations for a distro is not implemented"
+            )
```

Comment:
```suggestion
    def list_invalidations(self, dist_id):
        """
        Pagination is not yet implemented
        """
```

We can still return all objects without pagination - no reason to make the function unusable for people passing these arguments